src/bci_build/package/python.py

```diff
@@ -103,7 +105,7 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
 
     config_sh_script = ""
     if has_pipx:
-        config_sh_script += "install -d -o root -g root -m 0700 /root/.local/bin"
+        config_sh_script += "install -d -o root -g root -m 0700 /opt/pipx"
```

Comment:
that makes no sense anymore. the permissions would need to be updated.

Reply:
this entire line is not needed, pipx does create the home folder if it does not exist.